person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  attr_reader :id
+  attr_accessor :age, :name
```

Comment:
- [ ] [Optional][Highly recommended] It would be better professionally if you add Getters and Setters variables on the top of the `person` class before the `initialize` method.